core/src/main/java/apoc/nodes/Nodes.java

```diff
@@ -208,16 +210,31 @@ public Stream<NodeResult> get(
     public record DeletionLongResult(@Description("The number of deleted nodes.") Long value) {}
 
     @Procedure(name = "apoc.nodes.delete", mode = Mode.WRITE)
+    @QueryLanguageScope(scope = {QueryLanguage.CYPHER_5})
+    @Description("Deletes all `NODE` values with the given ids.")
+    public Stream<DeletionLongResult> deleteCypher5(
+            @Name(
+                            value = "nodes",
+                            description =
+                                    "The nodes to be deleted. Nodes can be of type `STRING` (elementId()), `INTEGER` (id()), `NODE`, or `LIST<STRING | INTEGER | NODE>`.")
+                    Object nodes,
+            @Name(value = "batchSize", description = "The number of node values to delete in a single batch.")
+                    long batchSize) {
+        return delete(nodes, batchSize);
+    }
+
+    @Procedure(name = "apoc.nodes.delete", mode = Mode.WRITE, deprecatedBy = "Cypher's `CALL {...} IN TRANSACTIONS`.")
```

Comment:
Missing a `@Deprecated`?

Reply:
The dynamic batch size can't be used in cypher, but feels like we can live with that.

Reply:
Yeah, we seem to have been okay with that from the initial CIT analysis, I just extrapolated from there 👀